pkg/api/norman/store/cluster/cluster_store.go

```diff
@@ -63,6 +65,7 @@ type Store struct {
 	NodeLister                    v3.NodeLister
 	ClusterLister                 v3.ClusterLister
 	DialerFactory                 dialer.Factory
+	ClusterClient                 dynamic.NamespaceableResourceInterface
```

Comment:
Swap to ResourceInterface, since a cluster is not in a namespace

Reply:
Done